### PR TITLE
Show query farmer sent sms

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

**Comment by admiral96** [comment]
Mingle Card #4031

File: app/views.py [hunk 1]
```diff
@@ -7,12 +7,13 @@ def index():
...
 
 
-@url.route('/<query>')
-def print_user_input(query):
-    if '=' in query:
-        query_container, query_value = query.split('=')
-        return 'Your query is {} which is equal to {}'.format(query_container, query_value)
-    return "You've entered an incorrect query. Please check and try again. You can try 'url=something relevant'".format(query), 400
+@url.route('/query')
+def print_user_input():
```
Comment (mknecht) [inline app/views.py]: please don't use the word `print` as in python it means dumping text to stdout.

File: test_url_runner.py [hunk 2]
```diff
@@ -17,13 +17,21 @@ def test_invalid_url_page_not_found(self):
         self.assertEqual(result.status_code, 404)
 
     def test_invalid_query(self):
-       result = self.client.get('/any data input')
-       self.assertEqual(result.status_code, 400)
+        result = self.client.get('/query?no_content')
+        self.assertEqual(result.status_code, 400)
+
+    def test_invalid_query_empty(self):
```
Comment (mknecht) [inline test_url_runner.py:23]: can you add a test case for empty content string? i.e. “content=” ?
